impeller/renderer/backend/vulkan/formats_vk.h

```diff
@@ -664,6 +570,51 @@ constexpr vk::ImageAspectFlags ToImageAspectFlags(PixelFormat format) {
   FML_UNREACHABLE();
 }
 
+struct SubpassCursorVK {
```

Comment:
Is this used anywhere?

Reply:
`PipelineVK::Create` uses a subpass cursor since it could technically still create those subpasses. But we don't call it. I'm backing it out.

Reply:
Done.